### DISCUSSION TITLE
🐣🐣🐣🐣What is your favorite emoji?🐣🐣🐣🐣

### DISCUSSION THREAD

**Discussion by ghost** [NAIST-SE/AttractingContributors#434] [General]
I like 🐣.
What about you?
Please send me Pull Requests.
I will send a Pull Request as well.

My fork:
https://github.com/techYamasaki/AttractingContributors-emoji

**Comment by aru1702** [comment]
😭 please don't judge me wrong

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407

**Reply by ghost** [reply]
It is so good!
Thank you😆

**Comment by XC1996114514** [comment]
My favourite emoj is 😅

So could you help me to filled the readme?
Here is my repository;
https://github.com/XianChen1996/AttractingContributors_type_in_your_favourite_footballteam

Thanks a lot for helping me!

**Reply by ghost** [reply]
It is so useful emoji😅.
Thank you.
I have filled your readme!!
